language/vm/vm-runtime/src/interpreter.rs

```diff
@@ -953,9 +953,16 @@ impl<'txn> Interpreter<'txn> {
     /// Given an `VMStatus` generate a core dump if the error is an `InvariantViolation`.
     fn maybe_core_dump(
         &self,
-        err: VMStatus,
+        mut err: VMStatus,
         current_frame: &Frame<'txn, FunctionRef<'txn>>,
     ) -> VMStatus {
+        // a verification error cannot happen at runtime so change it into an invariant violation.
```

Comment:
please let's think about this because I may be lying... ;)
essentially verification errors on publishing modules are not part of execution so the only way to get here should be when executing and some invariant was broken....

Reply:
Can we get here at runtime now that `load_and_verify_module_id` can generate a verification error?

Reply:
not sure what the question is :)
right now we can get here and the translation may not be perfect, but it would only happen for modules and we are not publishing modules. I planned to look at that next and see if a quick fix was worth it.
Overall though the idea is that we have done all cross module verification on publishing for modules and on verification for scripts so effectively the error is there but it should never happen. Or more precisely it should be an invariant violation.
Does it make sense?
Does it sound reasonable to you?

Reply:
Makes sense, thanks for clarifying!